src/MusicSrc/MusicDevice.c

```diff
@@ -468,6 +677,43 @@ static int NativeMidiInit(MusicDevice *dev, const unsigned int outputIndex, unsi
     ndev->dev.outputIndex = outputIndex;
     // send MIDI reset in case it was in a dirty state when we opened it
     NativeMidiReset(dev);
+#elif defined(__APPLE__)
+    // create Core MIDI client
+    if (ndev->midiClient) WARN("NativeMidiInit(): midiClient != 0");
+    const CFStringRef clientNameRef = CFStringCreateWithCString(NULL, "systemshockClient", kCFStringEncodingASCII);
```

Comment:
Another use of `CFSTR()`

    const CFStringRef clientNameRef = CFSTR("systemshockClient");
    const OSStatus clientResult = MIDIClientCreate(clientNameRef, NULL, NULL, &(ndev->midiClient));
    //CFRelease(clientNameRef); // Has no effect on CFStrings created with CFSTR